.gitignore

```diff
@@ -9,3 +9,5 @@ src_managed/
 project/boot/
 project/plugins/project/
 
+# IntelliJ
+.idea/*
```

Comment:
this should go into your global gitignore: https://help.github.com/en/articles/ignoring-files#create-a-global-gitignore :+1:

Reply:
Amazing! Thanks @BenFradet !